src/pages/index.tsx

```diff
@@ -1,3 +1,20 @@
+import Navigation from 'src/components/Navigation';
+import HeroSection from 'src/components/HeroSection';
```

Comment:
`HeroSection` would be in layout folder I think
```suggestion
import HeroSection from '@/layout/HeroSection';
```